cmd/buildah/manifest.go

```diff
@@ -639,13 +641,26 @@ func manifestPushCmd(c *cobra.Command, args []string, opts manifestPushOpts) err
 		return err
 	}
 
+	var manifestType string
+	if opts.format != "" {
+		switch opts.format {
+		case "oci":
+			manifestType = imgspecv1.MediaTypeImageIndex
+		case "v2s2", "v2s1", "docker":
```

Comment:
Accepting undocumented `docker`, fine. But why `v2s1`? To me that seems likely to mean the user doesn’t understand the values and we should reject instead of silently doing something else.

Reply:
That's mainly because we accept it as a value for `buildah push --format`.  I could go either way on whether or not we reject that value here, but the only documented valid values should be v2s2 and oci.

Reply:
Sure, that symmetry makes sense for `docker` and the others. Accepting the `v2s1` value and then silently doing `v2s2` is not quite the same thing — up to you, of course.

Reply:
Sure.  Dropped it as a recognized value.